common/src/View/GamesPreferencePane.h

```diff
@@ -45,6 +45,8 @@ namespace TrenchBroom {
         private:
             void createGui();
         private:
+            void showUserConfigDirClicked();
+            void showAppConfigDirClicked();
```

Comment:
This is not defined, right?

Reply:
Ah, sorry! I added that in and then changed my mind when I realized there were multiple resource paths involved. I'll remove it now.